### PR TITLE
Some small-ish MPC stuff

### PR DESCRIPTION
**Pull request by psychon**
This fixes a bug that made the code fail to detect disconnects. The rest is some other small-ish changes.

### CODE REVIEW

File: recipes/mpc.mdwn [hunk 1]
```diff
@@ -52,10 +53,15 @@ updates when the current MPD state changes.
         end
         mpd_widget.text = text
     end
+    local connection
     local function error_handler(err)
         mpd_widget:set_text("Error: " .. tostring(err))
+        -- Try a reconnect soon-ish
+        timer.start_new(60, function()
```
Comment (blueyed) [inline recipes/mpc.mdwn]: Isn't something shorter like 10 seconds better?

Comment (psychon) [inline recipes/mpc.mdwn]: Depends on how often you want to see the error message. ;-)
Best would be something like (almost-)exponential backoff, but that would be complicated. It's just an arbitrary number. Perhaps "1 minute" feels a bit less arbitrary than 10 seconds. I don't know and don't care much. If you want, I'll turn it into 10 seconds.

Comment (blueyed) [inline recipes/mpc.mdwn]: I just thought that a disconnect should not happen that often, and if so, it should not wait for a whole minute to reconnect.
Feel free to keep it.

**Comment by blueyed** [comment]
Needs to be debased.

**Comment by psychon** [comment]
Done (and changed the timeout to 10 seconds)

**Comment by blueyed** [comment]
Thanks!

**Pull request merged**